src/seals/diagnostics/random_trans.py

```diff
@@ -72,7 +72,10 @@ def __init__(
             n_states=n_states,
             rand_state=rand_gen,
         )
-        self.reward_weights = rand_gen.randn(observation_matrix.shape[-1])
+        
+        self.reward_weights = rand_gen.normal(
```

Comment:
loc=0 and scale=1 are defaults, can omit?

Reply:
yes we can